src/code/views/value-slider-view.tsx

```diff
@@ -485,4 +559,81 @@ export class SVGSliderView extends React.Component<SVGSliderViewProps, SVGSlider
       }
     }
   }
+
+  private toggleButtons() {
+    this.setState({showButtons: !this.state.showButtons});
+  }
+
+  private renderButtons() {
```

Comment:
These wiggle-toggle-buttons are complex enough to warrant their own component I think.